codebuild/CanaryWrapper_Classes.py

```diff
@@ -830,8 +830,7 @@ def monitor_loop_function(self, psutil_process : psutil.Process, time_passed=30)
         except Exception as e:
             self.print_message("[SnaptshotMonitor] ERROR - exception occurred checking metric alarms!")
             self.print_message("[SnaptshotMonitor] (Likely session credentials expired)")
-            self.had_internal_error = True
-            self.internal_error_reason = "Exception occurred checking metric alarms! Likely session credentials expired"
+            self.print_message("[SnaptshotMonitor] Not going to crash - just going to try again later")
```

Comment:
A little confused about the session credentials issue.  If providers are being used properly, expiration should not be a concern.

Reply:
Well just fyi, they are session-based credentials being used on EC2 (presumably imds-based).  But the provider will refresh because expiration is tracked as well.